internal/pkg/config/v1/context.go

```diff
@@ -191,32 +198,33 @@ func (c *Context) GetSuspensionStatus() *ccloudv1.SuspensionStatus {
 	return c.GetOrganization().GetSuspensionStatus()
 }
 
-func (c *Context) GetEnvironment() *ccloudv1.Account {
-	if auth := c.GetAuth(); auth != nil {
-		return auth.Account
+func (c *Context) GetCurrentEnvironment() string {
+	if c.CurrentEnvironment != "" {
+		return c.CurrentEnvironment
 	}
-	return nil
-}
 
-func (c *Context) SetEnvironment(environment *ccloudv1.Account) {
-	if c.GetAuth() == nil {
-		c.SetAuth(new(AuthConfig))
+	if auth := c.GetAuth(); auth != nil {
+		return auth.Account.GetId()
 	}
-	c.GetAuth().Account = environment
+
+	return ""
 }
 
-func (c *Context) GetEnvironments() []*ccloudv1.Account {
+func (c *Context) SetCurrentEnvironment(id string) {
```

Comment:
Same comment as above. Maybe we could just return at the top if `c == nil`

Reply:
If we're unable to set this, there's a bigger authentication problem that we should probably know about, so I'd opt to leaving this as is.

Reply:
I agree that we want to know about it. In that case I still think we should return an error instead of (potentially) a panic.